apps/shell-ui/scripts/stitch-flavors.mjs

```
@@ -0,0 +1,143 @@
+ *
+ * Steps: copy shell-ui-dev's hashed assets into shell-ui (hash-named — no
+ * collisions), extract each flavor's script/css tags from its emitted html,
+ * write the combined index.html with the picker + asset manifest.
+ */
+
+import fs from 'node:fs';
+import path from 'node:path';
+import { fileURLToPath } from 'node:url';
+
+const __dirname = path.dirname(fileURLToPath(import.meta.url));
+const BUILD_ROOT = process.env.ROCKETRIDE_BUILD_ROOT ?? path.resolve(__dirname, '../../../build');
```

ESLint

[error] 'process' is not defined.

(no-undef)